app/views/admin/loans/_tab.slim

```diff
@@ -1,4 +1,3 @@
-li role="presentation"
-  - classes = [*classes].join if defined? classes
```

Comment:
I don't know who added this in, but I changed something in parent template that makes this obsolete.